files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
```

Comment:
> Added, but I disagree with the etherpad and the justification is weak.

Please can you explain exactly why you think it is weak?  The two reasons given in the etherpad are:

1. this makes the reference clear to outsiders who don't know what bnc / bsc is
2. it makes it possible to open the bug with a single click when viewing git history

I really struggle to imagine how you could disagree with either of these.  For the first one, remember that this is a public repository, and the code relates to a fork of some upstream code which is still in use elsewhere.  So any comments about `bsc#XXXXXX` being sufficient in internal or openSUSE-specific tools are irrelevant.  And the second one is a simple fact.

>  For one side nearly all projects refers to bugs using the local prefix associated with namespace of the project, or even with a single number. For the other side our commits from other project do not follow this rule.

Again, I am not suggesting to drop the `bsc#XXXXX` format, so how is this relevant?  I am only saying that it is worth adding the full URL in *addition*, because this brings extra clarity and convenience for anyone who reads the PR and/or the git commits.  It's not *needed*, but it is a very nice courtesy which takes very little effort and makes other people's lives easier, so why not do it?  It follows the ethos of [this great article on Empathy and Pull Requests](https://slack.engineering/on-empathy-pull-requests-979e4257d158), which I recommend reading if you didn't already.

> No. Not really.

You *are* aware of it, even if you don't agree with it :-)

> When this policy was discussed? Who participate on it? When has been announced and communicated? I am not able to find this in the mailing list.

It was discussed a long time ago across several team workshops in Nuremberg, possibly before you joined the team, but I can't remember exactly when.